napari/_qt/layer_controls/qt_layer_controls_container.py

```diff
@@ -94,15 +94,12 @@ def _display(self, event):
         event : Event
             Event with the target layer at `event.item`.
         """
-        if event is None:
-            layer = None
-        else:
-            layer = event.item
+        active_layer = self.viewer.layers.active
```

Comment:
This line triggers recalculation on every function call. Why it's not using the cached value from the viewer?